### PR TITLE
[FIX] base: Preserve system user email

### PR DESCRIPTION
**Pull request by yajo**
After this migration, all system emails are sent by `__system__` instead of `admin`, so `__system__` should keep its email address to avoid altering the experience of users receiving emails.

The side effect will be that admin user will have no email address, which is not cool, so after migrating you should either update admin's email or remove him.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr
@Tecnativa TT19494

### CODE REVIEW

**Comment by pedrobaeza** [comment]
I have just discovered that we can't have 2 equal emails for partners linked to users (system gives an error), so let's go for switching the email and put in the other this fake email.

**Comment by yajo** [comment]
OK, fixed. It's less arbitrary and just:

1. Let `__system__` have the same email `admin` had.
1. Let the new admin have no email.

**Review by pedrobaeza — APPROVED** [review]
Waiting for @StefanRijnhart's approval as well

**Review by StefanRijnhart — APPROVED** [review]

**Comment by pedrobaeza** [comment]
FYI I have finally needed to assign an email to old admin user, or we got as well errors with some features requiring it: 87213fb8bab4a83237ce6d73b3c9924ee5ccfa96

Example error (in Spanish):

`El asociado 'x' está suscrito a una o más listas de correo. Se le debe asignar un email.`

**Pull request merged**
